.github/workflows/main-e2e.yml

```diff
@@ -59,6 +63,24 @@ jobs:
         continue-on-error: true
         run: npm list
 
+      - name: Pre-requisites for test run
```

Comment:
I was wondering if it makes sense to run this in a global before hook instead in a support file, so that each test does it's cleanup in isolation. That way we can cherry pick tests without needing to worry about running this one